tools/WebhookProcessor/github_webhook_processor.php

```diff
@@ -359,12 +359,32 @@ function check_dismiss_changelog_review($payload){
 				dismiss_review($payload, $R['id'], 'Changelog added/fixed.');
 }
 
+function check_webeditor($payload){
+	if(is_maintainer($payload, $payload['pull_request']['user']['login']))
+		return;
+	$githubgpgkeyid = '04AEE18F83AFDEB23';
+	$commits = json_decode(apisend($payload['pull_request']['commits_url']), TRUE);
+	foreach($commits as $commit) {
+		if(!$commit['commit']['verification']['verified'])
+			return false;
+		$sig = $commit['commit']['verification']['signature'];
+		$keyid = substr(bin2hex(base64_decode(substr(str_replace('\n', '', $sig),53,12))),0,-1);
```

Comment:
are you fucking denying based on signed commits?

holy shit dude, why

i don't want to spend 30 minutes dealing with git just to make a fucking grammar fix

Reply:
With even a modicum of knowledge it takes 5 minutes max to update your git locally.

Reply:
unless you don't have optic fiber connection, then it takes a day to update after any meaningful time passed because of half a gig size of the repo

Reply:
i have a lot of branches, a lot of unfinished projects, and a lot of in progress work I don't always want to commit

i gotta do a lot of different juggling for getting git up to speed

Reply:
It does exactly what I stated in the PR body. It checks if a commit was signed with GitHub's GPG key, which happens if, only if, and always if a commit is created in the webeditor. Try not to derail the convo.

Reply:
Does it know the difference between signed commits and commits auto signed by git?